### PR TITLE
Migrate Radio component to TypeScript

### PR DESCRIPTION
**Pull request by loonywizard**
https://kupibiletru.atlassian.net/browse/SITE-2954

### CODE REVIEW

**Comment by Kupibilet-System** [comment]
New version of package: `12.0.1-SITE-2954-migrate-ui-to-ts-ce552768`

**Comment by Kupibilet-System** [comment]
New version of package: `12.0.1-SITE-2954-migrate-ui-to-ts-58f90f4e`

**Comment by Kupibilet-System** [comment]
New version of package: `12.0.1-SITE-2954-migrate-ui-to-ts-362f78a0`

File: src/components/Radio/index.tsx [hunk 1]
```diff
@@ -1,67 +1,59 @@
...
-    className,
-    checked,
-    onChange,
+    disabled = false,
+    checked = false,
+    onChange = () => null,
   } = props
 
   return (
-    <RadioLabel disabled={disabled} className={className}>
```
Comment (zinovev) [inline src/components/Radio/index.tsx:58]: похоже, удаление className сломает [это](https://gitlab.amazon/frontend/kupibilet.ru/-/blob/SITE-2954/migrate-radio-to-ts/app/pages/Booking/BookingTwo/BookingForm/ServiceSection/components/ServiceControls.tsx#L51)

Comment (loonywizard) [reply src/components/Radio/index.tsx:58]: ой точно поправлю )))

**Comment by Kupibilet-System** [comment]
New version of package: `12.0.1-SITE-2954-migrate-ui-to-ts-473db356`

**Comment by Kupibilet-System** [comment]
New version of package: `12.1.0-SITE-2954-migrate-ui-to-ts-2d6249c6`

**Review by privetkometa — APPROVED** [review]

**Pull request merged**